### PR TITLE
tambah daftar developer

### PR DESCRIPTION
**Pull request by bayucakra20**
cmiiw

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ Terinspirasi oleh [sindresorhus/awesome](https://github.com/sindresorhus/awesome
 | Eko Prasetyo | ![foto](https://avatars.githubusercontent.com/u/24540408?s=75) | I'm programmer, I have no life | PHP | [GitHub](https://github.com/ekopradesga) · [Website](http://www.pradesga.com/) |
 | Disgan Adi | ![foto](https://avatars.githubusercontent.com/u/28666505?s=75) | Web Developer | HTML & CSS | [GitHub](https://github.com/izgan) |
 | Dwi Hujianto | ![foto](https://avatars0.githubusercontent.com/u/10525981?s=75) | Fullstack Developer | PHP,NodeJs | [GitHub](https://github.com/hujianto) · [Website](https://dwihujianto.com/) |
+| Bayu Cakra | ![foto](https://https://avatars.githubusercontent.com/u/13051657?s=75) | Web Developer | PHP | [Github](https://github.com/bayucakra20) | [Website](https://bayucakra.com) |
```
Comment (dikiaap) [inline README.md]: URL salah yu:
- https dobel
> https://https://avatars.githubusercontent.com/u/13051657?s=75

---
- fix table pls
> | [Github](https://github.com/bayucakra20) | [Website](https://bayucakra.com) |

Ga usah gawe `|` neng tengah. Markdown dadi rusak.

Dadi:
> | [GitHub](https://github.com/bayucakra20) · [Website](https://bayucakra.com) |

File: README.md [hunk 2]
```diff
@@ -25,6 +25,7 @@ Terinspirasi oleh [sindresorhus/awesome](https://github.com/sindresorhus/awesome
 | Eko Prasetyo | ![foto](https://avatars.githubusercontent.com/u/24540408?s=75) | I'm programmer, I have no life | PHP | [GitHub](https://github.com/ekopradesga) · [Website](http://www.pradesga.com/) |
 | Disgan Adi | ![foto](https://avatars.githubusercontent.com/u/28666505?s=75) | Web Developer | HTML & CSS | [GitHub](https://github.com/izgan) |
 | Dwi Hujianto | ![foto](https://avatars0.githubusercontent.com/u/10525981?s=75) | Fullstack Developer | PHP,NodeJs | [GitHub](https://github.com/hujianto) · [Website](https://dwihujianto.com/) |
+| Bayu Cakra | ![foto](https://avatars.githubusercontent.com/u/13051657?s=75) | Web Developer | PHP | [Github](https://github.com/bayucakra20) · [Facebook](https://facebook.com/bayucakraka) |
```
Comment (dikiaap) [inline README.md]: Kui GitHub yu, udu Github. :smile: 

Related: https://twitter.com/sindresorhus/status/735156430995591168

**Comment by bayucakra20** [comment]
wkwkwk, solah salah ae kat mau huhu

**Comment by dikiaap** [comment]
Wes yu.
https://github.com/ponorogodev/daftar-developer/commit/2a19638184c9bf0c5c06fd039a4f08830d7cbaba

Dan hai @ponorogodev di Telegram!
Tes komentar ini dari GitHub. wkwk

**Comment by dikiaap** [comment]
@dikiaap Tes komen dari Telegram!

**Pull request merged**